example/src/main/scala/CookieServerSide.scala

```diff
@@ -0,0 +1,30 @@
+import zhttp.http._
+import zhttp.service._
+import zio._
+
+import scala.concurrent.duration.DurationInt
+
+/**
+ * Example to make app using cookies
+ */
+object CookieServerSide extends App {
+  val cookie =
+    Cookie(name = "abc", content = "value", path = Some(Path("/cookie")), maxAge = Some(5 days))
```

Comment:
```suggestion
     Cookie("key", "value") @@ path(!! / "cookie") @@ maxAge(5 days)
```

Alternative way more "zioish" API.